en/lesson-08.md

```diff
@@ -57,98 +80,71 @@ source.
 \end{tabular}
 \end{document}
 ```
+<!-- {% endraw %} -->
 
+If a table column should contain a lot of text you will have issues to get that
+right with only `l`, `c`, and `r`. See what happens in the following example:
 
-## Adding lines
-
-A word of advice prior to introducing lines; lines should be used really
-sparsely in tables, and normally vertical ones often look unprofessional. In fact,
-for professional tables you shouldn't use any of the standard lines; instead you
-should skip this section and read about
-[`booktabs`](#tables-in-printing-quality-with-the-booktabs-package).
-
-Nevertheless, LaTeX has got you covered if you need lines. Vertical lines are part of
-the column specification and hence should go into the table preamble. To add a line
-between two columns just add `|` between their specification. Please note that
-vertical lines are only inserted if the cell to their left exists (except for a
-vertical line left of the first column). Note how in the following example the
-rightmost line will not appear in the last row.
-
+<!-- {% raw %} -->
 ```latex
 \documentclass{article}
 \usepackage{array}
 
 \begin{document}
-\begin{tabular}{|l|ll|}
-  Animal  & Food  & Size   \\
-  dog     & meat  & medium \\
-  horse   & hay   & large  \\
-  frog    & flies & small  \\
-  microbe & ???
+\begin{tabular}{cl}
+  Animal & Description \\
+  dog    & The dog is a member of the genus Canis, which forms part of the
+           wolf-like canids, and is the most widely abundant terrestrial
+           carnivore. \\
+  cat    & The cat is a domestic species of small carnivorous mammal. It is the
+           only domesticated species in the family Felidae and is often referred
+           to as the domestic cat to distinguish it from the wild members of the
+           family. \\
 \end{tabular}
 \end{document}
 ```
+<!-- {% endraw %} -->
 
-Horizontal lines belong to a `tabular`'s body. In LaTeX there are two different
-macros to add them, the first is `\hline` adding a line across the full width of
-a `tabular`, the second is `\cline` which can be used to draw lines covering
-only a specific range of columns.
-
-`\cline` takes a mandatory argument that specifies the columns affected. It has
-to be a range with a hyphen (`-`) as a separator between the first and the last
-column number. You can specify multiple `\cline`s in the same row. Both an
-`\hline` and a `\cline` have to be the first content of a new row (so directly
-following `\\` or another `\hline` or `\cline`).
-
+The issue is that the `l` type column typesets its contents in a single row at
+its natural width, meaning taking as much space as it takes, even if there is a
+page border in the way. To overcome this you can use the `p` column. This one
+typesets its contents as paragraphs with the width you specify as an argument
+and vertically aligns them at the top &ndash; which you'll want most of the
+time. Compare aboves outcome to the following:
```

Comment:
`aboves outcome` → `above's outcome`, or (more clearer, I think) `the previous example`